operations/observability/Observability.md

```diff
@@ -0,0 +1,336 @@
+Jan 25 05:00:09 redbull-pit prometheus[25680]: ts=2023-01-25T05:00:09.261Z caller=compact.go:519 level=info component=tsdb msg="write block" mint=1674612007>
+Jan 25 05:00:09 redbull-pit prometheus[25680]: ts=2023-01-25T05:00:09.263Z caller=head.go:840 level=info component=tsdb msg="Head GC completed" duration=1.7
+```
+
+#### Check Grafana service
+
+(`pit#`) Run the following command in order to show the status of the Grafana service.
+
+```bash
+systemctl status grafana.service
+```
+
+Example output of the service running properly on a PIT node:
+
+```text
+● grafana.service - Grafana
+     Loaded: loaded (/usr/lib/systemd/system/grafana.service; enabled; vendor preset: disabled)
+     Active: active (running) since Wed 2023-02-08 23:06:38 UTC; 4 days ago
+   Main PID: 82549 (conmon)
+      Tasks: 2
+     CGroup: /system.slice/grafana.service
+             ├─ 82540 /usr/bin/fuse-overlayfs -o lowerdir=/var/lib/containers/storage/overlay/l/UOU2YMGV3WT2CIASNIEDBIY6OK:/var/lib/containers/storage/overl>
+             └─ 82549 /usr/bin/conmon --api-version 1 -c f45f33ad520fb278776cf528dab1fdf619f0b1323e672d29d866f728ce8e2589 -u f45f33ad520fb278776cf528dab1fdf>
+
+Feb 08 23:06:38 redbull-pit grafana[82549]: logger=sqlstore t=2023-02-08T23:06:38.23+0000 lvl=info msg="Connecting to DB" dbtype=sqlite3
+Feb 08 23:06:38 redbull-pit grafana[82549]: logger=migrator t=2023-02-08T23:06:38.25+0000 lvl=info msg="Starting DB migrations"
+Feb 08 23:06:38 redbull-pit grafana[82549]: logger=migrator t=2023-02-08T23:06:38.26+0000 lvl=info msg="migrations completed" performed=0 skipped=393 durati>
+Feb 08 23:06:38 redbull-pit grafana[82549]: logger=plugin.manager t=2023-02-08T23:06:38.42+0000 lvl=info msg="Plugin registered" pluginId=input
+Feb 08 23:06:38 redbull-pit grafana[82549]: logger=query_data t=2023-02-08T23:06:38.43+0000 lvl=info msg="Query Service initialization"
+Feb 08 23:06:38 redbull-pit grafana[82549]: logger=live.push_http t=2023-02-08T23:06:38.44+0000 lvl=info msg="Live Push Gateway initialization"
+Feb 08 23:06:38 redbull-pit grafana[82549]: logger=grafanaStorageLogger t=2023-02-08T23:06:38.54+0000 lvl=info msg="storage starting"
+Feb 08 23:06:38 redbull-pit grafana[82549]: logger=ngalert t=2023-02-08T23:06:38.54+0000 lvl=info msg="warming cache for startup"
+Feb 08 23:06:38 redbull-pit grafana[82549]: logger=ngalert.multiorg.alertmanager t=2023-02-08T23:06:38.54+0000 lvl=info msg="starting MultiOrg Alertmanager"
+Feb 08 23:06:38 redbull-pit grafana[82549]: logger=http.server t=2023-02-08T23:06:38.55+0000 lvl=info msg="HTTP Server Listen" address=[::]:3000 protocol=http>
+```
+
+## IUF timing dashboard
+
+The IUF Timing dashboard provides timing information for installs and upgrades.
+
+### Timing dashboard features
+
+- Dynamic top-down and bottom-up dashboard to track install/upgrade status of any product, stage, and operation.
+- Dashboard calculates the duration for the install/upgrade of any product, stage, and operation.
```

Comment:
Both lines seem to be saying the same thing.  Can one be deleted?

Reply:
I see now that the second line mentions duration, so I guess they are different.  Given the similarities, perhaps this will summarize them both in a single line:
```A dynamic top-down and bottom-up dashboard that tracks install/upgrade status and duration for any product, stage, and operation.```